services/campaigns.service.ts

```
@@ -0,0 +1,34 @@
+        }
+      )
+      .then((res) => {
+        if (res.status === 200) {
+          Toaster.success("Campaign created successfully");
+        }
+      })
+      .catch((err) => {
+        Toaster.error(err.response.data.message);
+      });
+  };
+}
```

Biome

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)